KubernetesFile/dist-deployment-files/deployments/storage-manager-deployment.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    service: stomgr
+  name: stomgr
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      service: stomgr
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        service: stomgr
+    spec:
+      containers:
+      - image: scorpiobroker/scorpio:StorageManager_latest
```

Comment:
Define the postgress connection detail in deployment YAML instead of tightly bind in Storage manager in Java JAR package.
If user would like to use its own postgress than it would be not possible